### DISCUSSION TITLE
Object Versioning

### DISCUSSION THREAD

**Discussion by aurc** [fsouza/fake-gcs-server#1105] [General]
Hello,

Is there any plans to support object versioning? 

Thanks
Aurelio

**Comment by fsouza** [comment]
It is something we want to support, but we'd need a new filesystem-based backend that doesn't assume that the file path matches the object name. The memory backend supports it already.

There are no concrete plans to implement it though. Here's the issue tracking it: https://github.com/fsouza/fake-gcs-server/issues/744
